pkg/replication/replicator.go

```diff
@@ -178,9 +179,13 @@ func (r replicator) initializeReplicationWithCheck(
 	// To make a closed channel
 	close(replication.enactedCh)
 
-	err = replication.lockHosts(overrideLock, r.lockMessage)
-	if err != nil {
-		return nil, errCh, err
+	if skipLocking {
+		go replication.handleRenewalErrors(nil, nil)
```

Comment:
hey maybe this should be called something other than `handleRenewalErrors`, yeah?